src/utils.ts

```diff
@@ -210,10 +210,10 @@ export function setNextImageConfiguration(): void {
 
   // @ts-expect-error this is how Next.js seems to do
   process.env.__NEXT_IMAGE_OPTS = {
-    deviceSizes: config.images.deviceSizes,
-    imageSizes: config.images.imageSizes,
-    path: config.images.path,
-    loader: config.images.loader,
-    domains: config.images.domains,
+    deviceSizes: config.images?.deviceSizes,
+    imageSizes: config.images?.imageSizes,
+    path: config.images?.path,
+    loader: config.images?.loader,
+    domains: config.images?.domains,
```

Comment:
Would it be feasible covering this optional chaining logic branch with a test to get back to 100% test coverage?